### PR TITLE
Feat: Onboard Mimiciii dataset

### PR DESCRIPTION
**Pull request by Naveen130**
## Description

This is to onboard mimiciii dataset with 25 pipelines using Airflow v2 operators only.

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

Use the sections below based on what's applicable to your PR and delete the rest:

### Feature
- [ ] I'm adding or editing a feature
- [ ] I have updated the [`README`](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/README.md) accordingly
- [ ] I have added/revised tests for the feature

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/mimiciii` and nothing outside of that directory

### Code cleanup or refactoring
- [x] I'm refactoring or cleaning up some code

### CODE REVIEW

**Comment by Naveen130** [comment]
This PR includes 2 activities w.r.t. the dataset -
1. Removes mimicIII (uses Airflow v1 operator, not acceptable anymore)
2. Adds mimiciii (uses accepted Airflow v2 operator)

**Comment by adlersantos** [comment]
Use the BQ data transfer service. There are examples in the repo that you can just reuse:
https://github.com/GoogleCloudPlatform/public-datasets-pipelines/tree/main/datasets/scalable_open_source/pipelines/_images/bq_data_transfer

**Comment by adlersantos** [comment]
Also, please rename to mimic_iii

**Review by adlersantos — CHANGES_REQUESTED** [review]
Please use BQ data transfer and rename to mimic_iii

File: datasets/mimiciii/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+
+def main(
+    source_project, destination_project, source_dataset, destination_dataset, gcs_bucket
+):
+    logging.info(
+        "Fetching the source tables from bq. Each pipeline will be undergoing ETL"
+    )
+    source_table_names = fetch_source_tables(source_project, source_dataset)
+
```
Comment (nlarge-google) [inline datasets/mimiciii/pipelines/_images/run_csv_transform_kub/csv_transform.py:15]: Please remove extra lines

**Review by nlarge-google — CHANGES_REQUESTED** [review]
Please remove extra lines

**Pull request closed without merging**